Student_BCOs/Derek_Cao_BCO 000534.json

```diff
@@ -0,0 +1,175 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000534",
+    "checksum": "442ba25cf83b2e391b039a0326a711e3918e240bf7afe4e691b1299679fbc417",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3",
+        "version": "1.0.0",
+        "license": "https://creativecommons.org/licenses/by/4.0/",
+        "created": "2019-10-08T21:21:01.804217",
+        "modified": "2019-10-18T01:28:15.288271",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10/16/2019"
+            }
+        ],
```

Comment:
https://github.com/biocompute-objects/GW-SMHS-BIOC6223/pull/194/files/13ee6ba52702179503df76d0d5a8c51c35e49248#r336080331

@kee007ney

Reply:
@cderek7 It's really important that these have fully reworded descriptions stated in their entirety by you! Please write a few sentences of your own to describe the pipeline.

Reply:
Yes! I reword all the sentences already!